src/controllers/v1/entities.js

```diff
@@ -194,7 +194,8 @@ module.exports = class Entities extends Abstract {
 					req.query.entityType,
 					req.pageNo,
 					req.pageSize,
-					req?.query?.paginate?.toLowerCase() == 'true' ? true : false
+					req?.query?.paginate?.toLowerCase() == 'true' ? true : false,
+					req.query.language
```

Comment:
@MallanagoudaB entity controller has details, update apis as well. they should be also changed and tested